squeasel.c

```diff
@@ -4375,9 +4376,25 @@ static int set_ssl_option(struct sq_context *ctx) {
   }
 #endif // NO_SSL_DL
 
-  // Initialize SSL library
-  SSL_library_init();
-  SSL_load_error_strings();
+  // Initialize SSL library, unless the user has disabled this.
+  if (sq_strcasecmp(ctx->config[SSL_GLOBAL_INIT], "yes") == 0) {
```

Comment:
This doesn't really deal with the initialization race issue, right? Reason being that if we can't guarantee serialization of this section and a hosting application initializing SSL, we also can't guarantee that the hosting application has initialized SSL before we get to line 4399. i.e. the onus is on the hosting app to initialize SSL before initializing Squeasel.